.ci_support/osx_64_.yaml

```diff
@@ -5,17 +5,17 @@ boost_cpp:
 c_compiler:
 - clang
 c_compiler_version:
-- '12'
+- '13'
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 cxx_compiler:
 - clangxx
 cxx_compiler_version:
-- '12'
+- '13'
 ffmpeg:
-- '5'
+- '4.4'
```

Comment:
It seems like we are downgrading the `ffmpeg` version to upgrade the `compiler` version. @conda-forge/core Does this make sense?

Reply:
Yeah. I don't think we migrated to ffmoeg 5 yet. We tried. But saw many opened issues on important upstream packages we decided to hold off.

You should not be building for ffmoeg 5.0 today.

Reply:
Ok, and I see it only changed for `unix` so it should not be the reason why the windows builds started failing.

Reply:
I think windows is still on 4.3 since we can't build ffmoeg ourselves for windows.